util/write-package-info.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const path = require('path');
+
+const workingDirectory = process.argv[2];
+const destinationFile = process.argv[3];
+const packageJsonPath = path.join(process.cwd(), workingDirectory, 'package.json')
+const configDest = path.join(process.cwd(), workingDirectory, destinationFile);
+const packageJson = require(packageJsonPath);
+const packageInfo = {
+  name: packageJson.name,
+  version: packageJson.version
+};
+const output = 'export default ' + JSON.stringify(packageInfo, null, 2) + ';'
```

Comment:
nit: There is no `;` in our `okta-vue/src/*.js` files.

Reply:
This util script is at the root, so not part of lerna.  I can put together a PR to add eslint to the root, but we'd have to do testing to make sure it doesn't break what we just fixed for windows.